runtime/update_runtime.sh

```diff
@@ -22,7 +22,6 @@ if [[ ${VERSION::1} =~ [a-z] ]]; then
 fi
 
 OBS_PUSH=${OBS_PUSH:-false}
-STAGING=${STAGING:-true}
 OBS_RUNTIME_REPO=${OBS_RUNTIME_REPO:-home:clearcontainers:clear-containers-3-staging/cc-runtime}
 : ${OBS_APIURL:=""}
```

Comment:
what is this line for?

Reply:
Does the same as OBS_APIURL=${OBS_APIURL:-""}. I'll change it for better readability. Thanks!